### PR TITLE
Adding samples moved from azure-event-hubs-dotnet repo

### PR DESCRIPTION
**Pull request by sethmanheim**
## Description

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] **I have read the [contribution guidelines](./CONTRIBUTING.md).**
- [ ] Title of the pull request is clear and informative.
- [ ] If applicable, the code is properly documented.
- [ ] The code builds without any errors.

### CODE REVIEW

**Comment by sethmanheim** [comment]
cc @jtaubensee

File: samples/SampleSender/readme.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Get started sending messages to Event Hubs in .NET Core
+
+**Note**: This sample is also available as a tutorial [here](https://docs.microsoft.com/azure/event-hubs/event-hubs-dotnet-standard-getstarted-send).
+
+This sample shows how to write a .NET Core console application that sends a set of messages to an Event Hub. You can run the solution as-is, replacing the `EhConnectionString` and `EhEntityPath` strings with your Event Hub values, or you can follow the steps in the [tutorial](https://docs.microsoft.com/azure/event-hubs/event-hubs-dotnet-standard-getstarted-send) to create your own.
+
+## Prerequisites
+
+1. [Visual Studio 2015](http://www.visualstudio.com).
+2. [.NET Core Visual Studio 2015 tool](http://www.microsoft.com/net/core).
```
Comment (jtaubensee) [inline samples/SampleSender/readme.md]: tool -> tools/tooling?

File: samples/SampleEphReceiver/project.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  "buildOptions": {
+    "emitEntryPoint": true
+  },
+
+  "dependencies": {
+    "Microsoft.Azure.EventHubs.Processor": {
+      "target": "project"
+    },
+    "Microsoft.Azure.EventHubs": {
+      "target": "project"
```
Comment (jtaubensee) [inline samples/SampleEphReceiver/project.json]: This
```
    "Microsoft.Azure.EventHubs": {
       "target": "project"
```
will need to change to this:
```
"Microsoft.Azure.EventHubs": "1.0.0"
```

For both samples

File: samples/SampleSender/readme.md (hunk 1 above)
Comment (sethmanheim) [inline samples/SampleSender/readme.md]: Oops, thanks.

**Pull request merged**